java-util/src/main/java/io/druid/java/util/common/Numbers.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.java.util.common;
+
+public final class Numbers
+{
+  public static long parseLong(Object val)
```

Comment:
These should have some javadocs about their behavior covering at least:

- What types of Object are supported
- What happens if the Object is null (seems like it will throw ISE)
- What happens if the Object is unparseable (seems like it will throw either ISE or NumberFormatException, depending on type)

Consider also adding a comment about the relationship between this method and `GuavaUtils.tryParseLong`.

Reply:
Added.